File: zig/examples/console.zig
const std = @import("std");
const maple = @import("maple-sdk");

pub fn main() u8 {
    std.debug.print("Loaded maple-sdk from ZIG\n", .{});
    const result = maple.add(2, 4);
    std.debug.print("RESULT: {}\n", .{result});

    var foo = maple.stitchCreateMaple();
    std.debug.print("stitchCreateMaple: {}\n", .{foo});
    return 0;
}

File: zig/examples/console.c
#include <stdio.h>
#include <stdint.h>
#include "maple-sdk.h"

int main(int argc, char **argv) {
  printf("Loaded maple-sdk from C\n");
  int result = add(39, 3);
  printf("DLL result: %d\n", result);
  return 0;
}

